dev-aws/kafka-shared/iam_credentials.tf

```diff
@@ -5,8 +5,8 @@ resource "kafka_topic" "iam_credentials_v1" {
   config = {
     # retain 100MB on each partition
     "retention.bytes" = "104857600"
-    # keep data for 7 days
-    "retention.ms" = "604800000"
+    # keep data for 60 days
+    "retention.ms" = "5184000000"
```

Comment:
Retaining events for two months, but they are quite low-volume.

Reply:
Do we need the same in prod~?

Reply:
Oh boy, we only need it in prod 😅